x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -101,31 +94,19 @@ export function getServiceColumns({
       }),
       width: '40%',
       sortable: true,
-      render: (_, { serviceName, agentName, transactionType }) => (
+      render: (_, { serviceName, agentName }) => (
         <ToolTipWrapper>
           <EuiToolTip
             delay="long"
             content={formatString(serviceName)}
             id="service-name-tooltip"
             anchorClassName="apmServiceList__serviceNameTooltip"
           >
-            <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
-              {agentName && (
-                <EuiFlexItem grow={false}>
-                  <AgentIcon agentName={agentName} />
-                </EuiFlexItem>
-              )}
-              <EuiFlexItem className="apmServiceList__serviceNameContainer">
-                <AppLink
-                  data-test-subj="apmServiceListAppLink"
```

Comment:
Functional tests are failing because this `data-test-subj` is no longer part of the page.